postcss.config.js

```diff
@@ -13,14 +13,18 @@ const postCSSCustomMedia = require('postcss-custom-media')({
 const postCSSInlineSVG = require('postcss-inline-svg')()
 const colorFunction = require('postcss-color-function')
 
+const cssVariables = require('./src/config/css-variables')
```

Comment:
In my opinion, we can try to always use more declarative variable names, like in this case: `defaultCSSVariables`, does it make sense to you?

Reply:
For sure! I've changed this now 😄